CHANGES.txt

```diff
@@ -43,6 +43,9 @@ Features
   (Issue #1914)
 * Added setting ``SHOW_INDEX_PAGE_NAVIGATION`` which enables a basic
   page navigation for indexes. (Issue #2299)
+* Added settings ``DISABLE_INDEXES_PLUGIN_INDEX_AND_ATOM_FEED`` and
```

Comment:
PS. I would personally add new changelog entries at the top, but that’s pretty unimportant.